### PR TITLE
test

### PR DESCRIPTION
**Pull request by pjagielski**

### CODE REVIEW

File: src/main/java/pl/touk/sputnik/Test.java [hunk 1]
```diff
@@ -5,10 +5,11 @@
     public boolean test = false;
 
     public int method() {
+      String str = null;
         if (test) {
-            return 1;
+            return str.length( ) + 1;
```
Comment (sputnikci) [inline src/main/java/pl/touk/sputnik/Test.java]: \[Checkstyle\] ERROR: '\(' is followed by whitespace\.

Comment (sputnikci) [inline src/main/java/pl/touk/sputnik/Test.java]: \[Checkstyle\] ERROR: '\)' is preceded with whitespace\.

Comment (sputnikci) [inline src/main/java/pl/touk/sputnik/Test.java]: \[FindBugs\] WARNING: NP: Null pointer dereference of str in pl\.touk\.sputnik\.Test\.method\(\)

File: src/main/java/pl/touk/sputnik/Test.java [hunk 2]
```diff
@@ -5,10 +5,12 @@
     public boolean test = false;
 
     public int method() {
+      String str = null;
```
Comment (pjagielski) [inline src/main/java/pl/touk/sputnik/Test.java]: Nice change

Comment (pjagielski) [inline src/main/java/pl/touk/sputnik/Test.java]: NOT

File: src/main/java/pl/touk/sputnik/Test.java [hunk 3]
```diff
@@ -5,10 +5,12 @@
     public boolean test = false;
 
     public int method() {
+        String str = null;
         if (test) {
-            return 1;
+            return str.length( ) + 3;
```
Comment (sputnikci) [inline src/main/java/pl/touk/sputnik/Test.java]: \[Checkstyle\] ERROR: '\(' is followed by whitespace\.

Comment (sputnikci) [inline src/main/java/pl/touk/sputnik/Test.java]: \[Checkstyle\] ERROR: '\)' is preceded with whitespace\.

Comment (sputnikci) [inline src/main/java/pl/touk/sputnik/Test.java]: \[FindBugs\] WARNING: NP: Null pointer dereference of str in pl\.touk\.sputnik\.Test\.method\(\)

File: src/main/java/pl/touk/sputnik/Test.java [hunk 4]
```diff
@@ -5,10 +5,12 @@
     public boolean test = false;
 
     public int method() {
+        String str = null;
         if (test) {
-            return 1;
+            return str.length( ) + 4;
```
Comment (sputnikci) [inline src/main/java/pl/touk/sputnik/Test.java]: \[Checkstyle\] ERROR: '\(' is followed by whitespace\.

Comment (sputnikci) [inline src/main/java/pl/touk/sputnik/Test.java]: \[Checkstyle\] ERROR: '\)' is preceded with whitespace\.

Comment (sputnikci) [inline src/main/java/pl/touk/sputnik/Test.java]: \[FindBugs\] WARNING: NP: Null pointer dereference of str in pl\.touk\.sputnik\.Test\.method\(\)

Comment (sputnikci) [inline src/main/java/pl/touk/sputnik/Test.java:10]: [Checkstyle] ERROR: '(' is followed by whitespace.

Comment (sputnikci) [inline src/main/java/pl/touk/sputnik/Test.java:10]: [Checkstyle] ERROR: ')' is preceded with whitespace.

Comment (sputnikci) [inline src/main/java/pl/touk/sputnik/Test.java:10]: [FindBugs] WARNING: NP: Null pointer dereference of str in pl.touk.sputnik.Test.method()

**Pull request closed without merging**